### PR TITLE
add new table and new migration

### PR DESCRIPTION
**Pull request by Nazarcheck**

### CODE REVIEW

File: lesson5/app/models/diagnosis_item.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class DiagnosisItem < ApplicationRecord
+    belongs_to :physician
+    belongs_to :patient
+end
+
```
Comment (olehvavryniv) [inline lesson5/app/models/diagnosis_item.rb]: EOF problem

File: lesson5/app/models/physician.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
 class Physician < ApplicationRecord
     has_many :appointments
     has_many :patients, through: :appointments
+
```
Comment (olehvavryniv) [inline lesson5/app/models/physician.rb:5]: `has_many :diagnosis_items` also

File: lesson5/db/schema.rb [hunk 3]
```diff
@@ -25,16 +25,32 @@
...
+    t.bigint "patient_id"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["patient_id"], name: "index_diagnosis_items_on_patient_id"
+    t.index ["physician_id"], name: "index_diagnosis_items_on_physician_id"
+  end
+
   create_table "patients", force: :cascade do |t|
-    t.string "name"
+    t.string "first_name"
```
Comment (olehvavryniv) [inline lesson5/db/schema.rb]: Should be NOT NULL.

File: lesson5/db/schema.rb [hunk 4]
```diff
@@ -25,16 +25,32 @@
     t.index ["physician_id"], name: "index_appointments_on_physician_id"
   end
 
+  create_table "diagnosis_items", force: :cascade do |t|
+    t.string "content"
```
Comment (olehvavryniv) [inline lesson5/db/schema.rb]: Should be NOT NULL.

File: lesson5/db/schema.rb (hunk 3 above)
Comment (olehvavryniv) [reply lesson5/db/schema.rb]: Check other fields that should be NOT NULL.

**Review by olehvavryniv — APPROVED** [review]

**Review by Alice932 — APPROVED** [review]

**Pull request merged**